doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -139,38 +139,92 @@ The powersave level can be specified by using the [option]`radeon_powersave` opt
 For details, see link:http://www.x.org/wiki/RadeonFeature#KMS_Power_Management_Options[www.x.org]. Note that this plug-in is experimental and the option might change in future releases. 
 
 `bootloader`::
-Adds options to the kernel command line. This plug-in supports only the GRUB 2 boot loader.
+Adds options to the kernel command line. This plug-in supports the GRUB 2 boot loader and the Boot Loader Specification (BLS).
 +
 Customized non-standard location of the GRUB 2 configuration file can be specified by the [option]`grub2_cfg_file` option.
 +
 The kernel options are added to the current GRUB configuration and its templates. The system needs to be rebooted for the kernel options to take effect.
 +
-Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file.
+Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file and grub environment files.
 +
 The kernel options can be specified by the following syntax:
```

Comment:
With the new additions, this list item has grown to be quite long, both absolutely and in comparison to the other list items around.

This particular AsciiDoc file is intended as a reference listing the available Tuned plug-ins. The additions really explain the usage of the `bootloader` plug-in, rather than simply saying that this plug-in is available.

For this reason, I suggest that we split the content starting with this line into a new file, such as a procedure called "Editing boot loader options in a Tuned profile" or similar, or maybe a concept module focused just on the boot loader syntax and examples.

I leave it to the other writers to decide the best solution here.

Reply:
Could you or other writer provide patch or PR for it? I am really not good at AsciiDoc.